gulpfile.js

```diff
@@ -23,6 +25,10 @@ var browserify = require('browserify');
 var reactify = require('reactify');
 var uglify = require('gulp-uglify');
 
+// Flags.
+var isProduction = (argv.p === undefined) ? false : true;
```

Comment:
what.. how does this work? -p flag? I think uglify should just be its own task maybe? and append it to production